src/es/administration/from_scratch.md

```diff
@@ -0,0 +1,395 @@
+# Instalar desde Cero
+
+> ⚠️ **Descargo de responsabilidad:** este método de instalación no está recomendado por los desarrolladores de Lemmy. Si tienes algún problema, debes resolverlo tú mismo o preguntar a los respectivos autores. Si observas algún fallo de Lemmy en una instancia instalada de este modo, por favor, menciónalo en el informe de fallos.
+
+>  ℹ️ Este documento no cuenta con traducción al carecer de soporte y ser de un caracter experimental.
+
+## Installing Lemmy from source code
+
+Instructions for installing Lemmy natively, without relyng on Docker. Originally posted on [Some resources on setting Lemmy up from source - Lemmy dot C.A.](https://lemmy.ca/post/1066) **The current transcription has been adapted to improve readability**.
```

Comment:
Same, I agree with @Nutomic here, it'd be better to leave a placeholder like TODO or something.

Reply:
Done. The translation of  all files is complete :smiley: